### DISCUSSION TITLE
Elephant's Foot Compensation

### DISCUSSION THREAD

**Discussion by Garryb123** [supermerill/SuperSlicer#2662] [General]
I love your work. Do you plan to implement Elephant's Foot Compensation like Cura and PrusaSlicer have?

**Comment by neophyl** [comment]
Its already there, its been there forever, just like PS.  Print Settings>Slicing>XY First Layer compensation.

One thing you do have to take into consideration though is it is different to Prusa Slicer.  Firstly the value is different to PS.  PS uses 0.1 to bring the layers inwards by 0.1mm.  In SuSi 0.1 will move the layers OUTWARDS.  -0.1 will move them INWARDS.  Also when you look you will there is also a height in layers value.  This allows you to have the adjustment not just on the first layer.

The more General XY compensation is is in addition to the first layer compensation, both values work together.  So you can shrink the whole model in XY by -0.1 and then further shrink the bottom x layers by -0.2 using the XY First layer compensation, (or push those layers back out by using a positive value).

Generally if a setting is in PS then it will exist in SuSi.  It might be named differently or located in a different screen though and as with the Elephants foot value the value itself might be treated differently so you have to be careful switching between PS and SuSi. 
SuSi has far more settings, that can be both a good thing and a bad one.  It allows more control, but it allows more control.  You the user are expected to know what you are doing :)

If you don't, well that's what here is for.

**Reply by Garryb123** [reply]
Thanks a lot for the full explanation. Obviously I didn't look carefully enough; coming from using KISSlicer most of printing life I was a little overwhelmed with the number of settings. But I like this! SuperSlicer's implementation makes much more sense than PrusaSlicer's - IMO positive or negative values should reflect real-world direction of the part in XY. And being able to apply the compensation over multiple layers if needed is really good. I'm using SuperSlicer (leaning more to it) and PrusaSlicer more and more given KISSlicer may not be developed going forward. I've tried a lot but just can't love Cura. I'm glad SuperSlicer is available. Best regards ..
